riscv/processor.cc

```diff
@@ -330,9 +330,10 @@ isa_parser_t::isa_parser_t(const char* str)
       extension_table[EXT_ZDINX] = true;
       extension_table[EXT_ZFINX] = true;
     } else if (ext_str == "zqinx") {
-      extension_table[EXT_ZQINX] = true; //not work currently
+      extension_table[EXT_ZQINX] = true;
       extension_table[EXT_ZDINX] = true;
       extension_table[EXT_ZFINX] = true;
+      bad_isa_string(str, "'ZQinx' is not supported currently");
```

Comment:
Just delete all of this code.  The standard error handling is fine; no need for a Zqinx-specific message.